dashboard/app/controllers/application_controller.rb

```diff
@@ -137,7 +146,13 @@ def prevent_caching
     :parent_email_preference_opt_in,
     :parent_email_preference_email,
     school_info_attributes: SCHOOL_INFO_ATTRIBUTES,
-  ].freeze
+  ]
+
+  if rack_env?(:test) || rack_env?(:development)
+    (PERMITTED_USER_FIELDS << UI_TEST_ATTRIBUTES).flatten!
+  end
```

Comment:
Very small nit in how this is written: IMO it'd be more rubyish and concise to say
```suggestion
PERMITTED_USER_FIELDS.push(*UI_TEST_ATTRIBUTES) if %i(test development).include? rack_env
```